index.html

```diff
@@ -0,0 +1,1447 @@
+                        <h2>‘The Voice’ 2019 Top 4 Finale Performances & Predictions</h2>
+                    </div>
+        </div>
+
+
+        <div class="box14-divs">
+                <div class="box14-divs-left">
+                        <img alt="read-more" src="images/box14img.png">
+                    </div>
+    
+                    <div class="box14-divs-right">
+                        <h2>‘The Voice’ 2019 Top 4 Finale Performances & Predictions</h2>
+                    </div>
+        </div>
+
+
+        <div class="box14-divs">
+                <div class="box14-divs-left">
+                        <img alt="read-more" src="images/box14img.png">
+                    </div>
+    
+                    <div class="box14-divs-right">
+                        <h2>‘The Voice’ 2019 Top 4 Finale Performances & Predictions</h2>
+                    </div>
+        </div>
+
+    </div>
+
+</div>
+
+
+</div>
+
+</div>
+
+
+
+<!-- 15 - fifteenth container - footer -->
+
+<div class="margin-con">
+<footer>
+
+<div class="container-fluid fifteenth">
+
+
+<div class="footer-up d-md-flex"> 
+
+   <div class="footer-up-left d-flex">
+                <div class="wrap-up-left-l">
+
+            <div class="footer-up-left-lup"></div>
+
+            <div class="footer-up-left-ldown">
+            <img src="images/logon.png">  </div></div>
+
+<div class="wrap-up-left-r">
+            <div class="footer-up-left-rup"> </div>
+
+            <div class="footer-up-left-rdown">
+
+             <p> © 2019 NEWSWEEK </p>  
+
+            </div>
+
+        </div>
+
+            </div>
+
+   <div class="footer-up-right">
+        <div class="icons-social text-md-right">
+
+                        <i class="f1 fab fa-facebook-square"></i>
+                        <i class="f1 fab fa-twitter"></i>
+                        <i class="f1 fab fa-instagram"></i>
+                        <i class="f1 fab fa-tumblr"></i>
+                        <i class="f1 fab fa-linkedin"></i>
+
+                                </div>
+                        </div>
+
+</div>
+
+<div class="footer-down d-flex"> 
+                <div class="footer-down-left">
+
+                        <navbar>
+                        <a>About Us</a>
+                        <a>About Us</a>
+                        <a>About Us</a>
+                        <a>About Us</a>
+                        <a>About Us</a>
+                        <a>About Us</a>
+                        <a>About Us</a>
+                        <a>About Us</a>
+                        <a>About Us</a>
+                        <a>About Us</a>
+                        <a>About Us</a>
+                        <a>About Us</a>
+                        <a>About Us</a>
+                        <a>About Us</a>
+
+                        </navbar>
+            
+                </div>
+            
+                <div class="footer-down-right text-center">
+                                <navbar>
```

Comment:
Again, nav is better but also, if you are doing it like this, best not to have it inside the footer. 

Best to use `div` for the footer as opposed to `navbar`

Reply:
noted